src/NuGet.Core/NuGet.Build.Tasks.Pack/Strings.resx

```diff
@@ -163,7 +163,7 @@
     <value>Invalid target framework for the file '{0}'.</value>
   </data>
   <data name="IsPackableFalseError" xml:space="preserve">
-    <value>Cannot generate a package from a project which has IsPackable property set to false.</value>
+    <value>Skipping package generation from this project because packaging has been disabled. Add &lt;IsPackable&gt;true&lt;/IsPackable&gt; to the project file to allow producing a package from this project.</value>
```

Comment:
"Skipping" would seem more appropriate if this was a warning. Considering this is an error, what do you think of:

```suggestion
    <value>This project cannot be packaged because packaging has been disabled. Add &lt;IsPackable&gt;true&lt;/IsPackable&gt; to the project file to allow producing a package from this project.</value>
```

Reply:
Great suggestion by the way. I've ran into this issue before and it took me way longer than I'd like to admit to figure it out :)